alarm_central_station_receiver/main.py

```diff
@@ -232,6 +241,10 @@ def main():
             sys.stdout if args.no_fork else None))
     context.signal_map = {signal.SIGTERM: sigcleanup_handler,
                           signal.SIGINT: sigcleanup_handler}
+
+    if args.notification_test:
```

Comment:
Can you move this to right below the 'if args.create_config' check?

Reply:
It won't until after the **initialize(args.config_path)** call has populated the config, I could put it directly after that call.